README.md

```diff
@@ -126,6 +126,19 @@ usage: que [options] [file/to/require] ...
 
 You may need to pass que a file path to require so that it can load your app. Que will automatically load `config/environment.rb` if it exists, so you shouldn't need an argument if you're using Rails.
 
+## Usage with Active Job
```

Comment:
Can't we just use the "Additional Rails-specific Setup" for that?

Reply:
Done! Is that what you had in mind?

Reply:
Yes! 😄